pkg/registration/hub/importer/options/options.go

```diff
@@ -1,16 +1,30 @@
 package options
 
-import "github.com/spf13/pflag"
+import (
+	"fmt"
+
+	"github.com/spf13/pflag"
+	"k8s.io/client-go/kubernetes"
+
+	"open-cluster-management.io/ocm/pkg/registration/hub/importer"
+)
 
 type Options struct {
-	APIServerURL string
-	AgentImage   string
-	BootstrapSA  string
+	APIServerURL      string
+	AgentImage        string
+	BootstrapSA       string
+	ImporterRenderers []string
 }
 
+const (
+	RenderFromConfigSecret string = "render-from-config-secret"
```

Comment:
we need to add some description for each for this const

Reply:
added